### DISCUSSION TITLE
C++17 compatible release/branch

### DISCUSSION THREAD

**Discussion by ker2x** [openframeworks/openFrameworks#7312] [Q&A] [answered]
Hi,

i really really like openframeworks, but the incompabilities with C++17 make it a real big problem for me (can't use intel compiler and oneAPI). Is there a (possibly unstable) release/tag/branch that works with C++17, on windows ? Or perhaps a manual fix/workaround for it ? 

i found a few old discussion/issues about it, but they're old.

thank you.

**Comment by ofTheo** [comment, marked as answer]
Hi @ker2x 

The nightly builds ( at the bottom of the download page ) are all C++17: 
https://openframeworks.cc/download/

The next release will be C++17 but the nightlies are pretty close at this point ( to release ). 

Let us know if you have any issues getting it working.

**Reply by ker2x** [reply]
thank you :)
